### ISSUE TITLE
Website Accessibility

### ISSUE THREAD

**Issue by AlanBridgeman** [asikkenga/devClub#1]
After looking at some of the HTML code I noticed that the alt texts on some of the inages were lacking. Particularly those that only say open this could be xonfusing for someone using a screen reader. You also dosn't set the lang attrivute. I also noticed that in in the form area you didn't include any labels for the inputs just placeholders which tcoud be confussing for users thar are older or those with disabilities. I'm sure if I look hard enough I could find other issues as well 

I highly recommend checking out http:///w3c.org/wai for information and resources related to the Web Content Accessibility Guideline (WCAG 2.0) AA compliance is industry standard but AAA wouldn't be bad thing either. The WAVE evaluation tool can be a good place to start as well.

Hope this helps!

**Comment by AlanBridgeman** [comment]
Apparently I missed the lang attribute my bad there.

**Comment by asikkenga** [comment]
"Open" is only a placement right now until we fill the positions and name
the images appropriately.   The code and everything is still rough and
started out as a template. I havnt had time to cleab things up yet, only
throw it together. Feedback is very helpful though! Thank you.

**Comment by ratstache** [comment]
I've copied over the conversation to the umdevclub organization repo and logged a new issue. If someone with authorization could close this that would be great!
